mmcv/runner/base_runner.py

```diff
@@ -103,6 +103,7 @@ def __init__(self,
         self.optimizer = optimizer
         self.logger = logger
         self.meta = meta
+        self.should_stop = False
```

Comment:
Suggest adding a comment to describe the function of the flag `self.should_stop`